example/azure/provider.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_providers {
+    autobotai = {
+      source  = "shunyeka/autobot/autobot"
+    }
+  }
+}
+
+provider "autobotai" {
+  apikey = var.apikey
```

Comment:
We need to ask user 2 variables. apikey and url.

url = "api.autobot.live" or "api-test.autobot.live" etc.